h/models/helpers.py

```diff
@@ -0,0 +1,4 @@
+def repr_(obj, attrs):
+    class_name = type(obj).__name__
+    attrs = {attrname: getattr(obj, attrname) for attrname in attrs}
+    return f"{class_name}({', '.join(f'{name}={value!r}' for name, value in attrs.items())})"
```

Comment:
Used this code in a couple of places now so moved it to a helper function.

Longer-term I think we should move all h's models to [SQLAlchemy's dataclasses support](https://docs.sqlalchemy.org/en/20/orm/dataclasses.html) (which we use in Via) then we won't need this.